src/tabs/tab.directive.ts

```diff
@@ -12,13 +12,21 @@ import {
 } from '@angular/core';
 import { TabsetComponent } from './tabset.component';
 
-@Directive({ selector: 'tab, [tab]' })
+let id = 0;
+
+@Directive({
+  selector: 'tab, [tab]',
+  host: {
+    role: 'tabpanel'
+  }
+})
 export class TabDirective implements OnInit, OnDestroy {
   /** tab header text */
   @Input() heading: string;
   /** tab id. The same id with suffix '-link' will be added to the corresponding &lt;li&gt; element  */
+  @HostBinding('attr.aria-labelledby')
```

Comment:
aria-labeledby must have a reference to element that triggers the tab, so it should be an id of `<a>` element (just tab id with `-link` suffix)